config/initializers/cors.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'stage-ttt-web-client.herokuapp.com'
```

Comment:
Am I inferring correctly that you are whitelisting the list of URLs you are allowing to access the API? If so, nice touch.

Reply:
yup, only requests from stage-ttt-web-client.herokuapp.com will be allowed.